finagle-core/src/test/scala/com/twitter/finagle/util/LoadServiceTest.scala

```diff
@@ -94,6 +95,12 @@ class LoadServiceTest extends FunSuite with MockitoSugar {
     executor.shutdown()
   }
 
+  test("LoadService should ignore packages according to ignoredPaths GlobalFlag") {
```

Comment:
I suspected it could have been something of that sort so i had commented out all previous tests and ran only the one i wrote but with the same results. Regardless, i would need to look into this and see how i can work around it (probably trying to use the methods you mentioned).